src/main/com/fulcrologic/rad/pathom_common.clj

```diff
@@ -1,17 +1,15 @@
 (ns com.fulcrologic.rad.pathom-common
   "Helper functions that are used by both Pathom 2 and 3 support."
   (:require
-    [clojure.pprint :refer [pprint]]
-    [clojure.walk :as walk]
-    [com.fulcrologic.rad.attributes :as attr]
-    [edn-query-language.core :as eql]
-    [taoensso.timbre :as log]))
+   [clojure.walk :as walk]
+   [taoensso.timbre :as log]))
 
 (defn remove-omissions
   "Replaces black-listed keys from tx with :com.fulcrologic.rad.pathom/omitted, meant for logging tx's
   without logging sensitive details like passwords."
   [config tx]
-  (let [sensitive-keys (get-in config [:com.fulcrologic.rad.pathom/config :sensitive-keys] #{})]
+  (let [sensitive-keys (conj (:sensitive-keys config #{})
```

Comment:
This is a breaking change that moves where the config parameter is placed.

Reply:
Or was this the layering problem?

Reply:
That is the layering problem. The config parameter is already unwrapped earlier when creating the parser. This PR works with the location of the config in fulcro-rad-demo unchanged